Foundation/NSDecimalNumber.swift

```diff
@@ -51,6 +51,10 @@ public class NSDecimalNumber : NSNumber {
         NSUnimplemented()
     }
     
+    public required convenience init(bytes buffer: UnsafePointer<Void>, objCType: UnsafePointer<Int8>) {
+        NSUnimplemented()
```

Comment:
yea; the objc version does this
`Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** initialization method -initWithBytes:objCType: cannot be sent to an abstract object of class NSDecimalNumberPlaceholder: Create a concrete instance!'`
when calling it via `NSDecimalNumber *dcn = [[NSDecimalNumber alloc] initWithBytes:&dcv objCType:@encode(NSDecimal)];`